spec/lib/email_processor_spec.rb

```diff
@@ -90,5 +89,43 @@
 
       expect(ticket.messages.length).to eq(2)
     end
+
+    it 'should assign forwarded emails to the original sender' do
+      student
+      body = <<~EXAMPLE
+        Example email test\r\n\r\n---------- Forwarded message ---------\r\nFrom:
+        Student <student@email.com>\r\nDate: Mon, Apr 8, 2019 at 3:42 PM\r\n
+        Subject: Help!\r\nTo: <staff@email.com>\r\n\r\n\r\nHelp message\r\n
```

Comment:
If `staff@email.com` is also a user, it'll probably be the earlier one in the database, so we need to be sure that we don't match that as the sender.